Makefile

```diff
@@ -38,13 +38,13 @@ internal_image_registry:=image-registry.openshift-image-registry.svc:5000
 # corresponding image stream inside that namespace. If the namespace doesn't
 # exist the push fails. This doesn't apply when the image is pushed to a public
 # repository, like `docker.io` or `quay.io`.
-image_repository:=$(namespace)/ocm-example-services
+image_repository:=$(namespace)/rh-trex
 
 # Database connection details
-db_name:=ocmexample
-db_host=ocm-ex-service-db.$(namespace)
+db_name:=rhtrex
+db_host=rh-trex-db.$(namespace)
 db_port=5432
-db_user:=ocm_example_service
+db_user:=rh_trex
```

Comment:
recommend making this a hypen so that the token becomes `rh-trex` or just `trex` everywhere.

Then when we need to make a new instance, we search/replace a minimal number of tokens.